.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version-file: ".nvmrc"
-      - run: npm install -g yarn@^1.19.0 && yarn install
+      - run: npm install -g yarn@^1.22.22 && yarn install
```

Comment:
The Github CI runners already have 1.22.22 pre-installed, so I think it would be better to drop the CI changes in this PR and cherry-pick commit c38759c09 instead.

Reply:
if it is preinstalled then this shouldn't make any changes